src/schema/v2/me/update_me_mutation.ts

```diff
@@ -260,3 +260,21 @@ export default mutationWithClientMutationId<any, any, ResolverContext>({
     }
   },
 })
+
+export const computeImageSources = (externalImageUrls) => {
```

Comment:
We don't need support for remote image URLs here and it's maybe better if this mutation uses it's own `computeImageSources` function.